### DISCUSSION TITLE
[FEATURE] 泛域名优化建议

### DISCUSSION THREAD

**Discussion by cktk** [1Panel-dev/1Panel#7163] [Ideas]
### 1Panel 版本

v1.10.20-lts

### 请描述您的需求或者改进建议

例如我想配置一个  `*.baidu.com`
我在主站写*.baidu.com 会导致网站无法访问 
首先是目录 不允许有* 只能改成  .baidu.com  然后点击删除网站  确认弹框中显示请输入

目前解决方法 主站写www.baidu.com  其他域名中输入 *.baidu.com  

我倒是希望能写*.baidu.com 当做主站 当然 这个不影响使用

**Comment by zhengkunwang223** [comment]
.baidu.com ? 这种吗 不是 *.baidu.com ?

**Comment by cktk** [comment]
` *.baidu.com`  刚才好像是被转义了  
这样写会提示错误 所以目录只能改一下  
<img width="984" alt="image" src="https://github.com/user-attachments/assets/ba1f4849-0891-48e5-8fcd-5b580963151f">
修改后的代号  这样会导致网站无法访问  正常来说 我访问  abc.baidu.com 应该是可以访问的 但是我一直是404
<img width="787" alt="image" src="https://github.com/user-attachments/assets/e26c7d1b-651e-43a3-b9fe-c262b3064eed">

**Comment by zhengkunwang223** [comment]
> ` *.baidu.com` 刚才好像是被转义了 这样写会提示错误 所以目录只能改一下 <img alt="image" width="984" src="https://private-user-images.githubusercontent.com/49108554/386463578-ba1f4849-0891-48e5-8fcd-5b580963151f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE2NDg1MTcsIm5iZiI6MTczMTY0ODIxNywicGF0aCI6Ii80OTEwODU1NC8zODY0NjM1NzgtYmExZjQ4NDktMDg5MS00OGU1LThmY2QtNWI1ODA5NjMxNTFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE1VDA1MjMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEzZDIwOGY3NGEwOTI1ZTk5NzIyMTllNTMyMzQxNjMwNjMwMTkwNWM1OWY4ZTI2OTA0MmQyNzczMGM4ZDI3MGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.108B7S_Fe8LXtKe2D9qU9ecZ7Lz_B_k1gfoFkhhC2nk"> 修改后的代号 这样会导致网站无法访问 正常来说 我访问 abc.baidu.com 应该是可以访问的 但是我一直是404 <img alt="image" width="787" src="https://private-user-images.githubusercontent.com/49108554/386464585-e26c7d1b-651e-43a3-b9fe-c262b3064eed.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE2NDg1MTcsIm5iZiI6MTczMTY0ODIxNywicGF0aCI6Ii80OTEwODU1NC8zODY0NjQ1ODUtZTI2YzdkMWItNjUxZS00M2EzLWI5ZmUtYzI2MmIzMDY0ZWVkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE1VDA1MjMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYwYjgxMTdhMGI4OGI4MDg1NmRkZDhlODc5NmRkNzNhMzdmODgyOTFlZjI3ODAxY2Q2Mjc1MGM5NzBhZjc2ODUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.eWAESV1CuBAyYVRT3Fh7CtC4q37g7C04_w7xh5ER0ew">

代号是为了创建网站目录用的 所以不能以 . 开头 ，删除的时候用的是主域名 不是代号

**Comment by cktk** [comment]
所以这个不允许泛域名作为主站吗

**Comment by zhengkunwang223** [comment]
> 所以这个不允许泛域名作为主站吗

允许

**Comment by ChishFoxcat** [comment]
代号仅仅是创建网站时文件夹的名称，输入框下对应有提示
可以另起代号代替，不会影响域名
